include/cutlass/numeric_conversion.h

```diff
@@ -2352,8 +2352,9 @@ struct FastNumericArrayConverter {
   static result_type convert(source_type const &s) {
     result_type result;
```

Comment:
Thank you for the PR.  Could we just remove this line?

Reply:
Thanks for your time to review, sure, I will follow it.